ui/App/DesignSystemGuideModal.svelte

```diff
@@ -195,11 +197,16 @@
             on:select={ev => theme.set(ev.detail)} />
         </Tooltip>
         <Tooltip value="Primary color" position="bottom">
-          <SegmentedControl
-            style="background-color: var(--color-background); margin-right: 2rem;"
-            active={$primaryColor}
-            options={primaryColorOptions}
-            on:select={ev => primaryColor.set(ev.detail)} />
+          <div style="display: flex;">
+            <SegmentedControl
+              style="background-color: var(--color-background); margin-right: 1rem;"
+              active={$primaryColor}
+              options={primaryColorOptions}
+              on:select={ev => primaryColor.set(ev.detail)} />
+            <ColorInput
+              style="margin-right: 2rem;"
+              bind:value={$primaryColorHex} />
```

Comment:
Can we replace this with `<SegmentedColorPicker>` as well?

Reply:
yep